src/modules/aws_ec2_metadata/index.ts

```diff
@@ -118,4 +118,29 @@ class AwsEc2MetadataModule extends ModuleBase {
     super.init();
   }
 }
+
+/**
+ * ## Code examples
+ *
+ *
```

Comment:
any reason for the extra space?

Reply:
formatting... otherwise lint doesn't pass